src/pages/SarStake/StakeStat/StakeStat.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useMemo } from 'react'
+import { CurrencyLogo, Text, useSarStakeInfo, useSarPositions } from '@pangolindex/components'
+import { BigNumber } from '@ethersproject/bignumber'
+import numeral from 'numeral'
+import Stat from 'src/components/Stat'
+import { PNG } from 'src/constants/tokens'
+import { useChainId } from 'src/hooks'
+import { DestkopDetails, MobileDetails, Title, Wrapper } from './styleds'
+import { formatEther } from 'ethers/lib/utils'
+
+const StakeStat: React.FC = () => {
+  const chainId = useChainId()
+  const { APR, totalStaked } = useSarStakeInfo()
```

Comment:
lets make this to `apr`
```suggestion
  const { apr, totalStaked } = useSarStakeInfo()
```